src/lib/OpenApi.ts

```
@@ -210,6 +210,73 @@
+              tags.push(tag)
+            }
+          })
+        }
+      }
+      return tags
+    }, [])
+  }
+
+  function getPathsByTags(tags: string | string[]) {
+    if (typeof tags === 'string') {
+      tags = [tags]
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)